src/layout/FooterMarketing.scss

```diff
@@ -0,0 +1,42 @@
+@import '~@redhat-cloud-services/frontend-components-utilities/styles/_mixins';
+@import '~@patternfly/patternfly/sass-utilities/_all';
+
+.ins-c-footer-marketing {
+  --pf-c-page--section--m-limit-width--MaxWidth: #{pf-size-prem(1140px)};
+  --pf-c-page__main-section--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-page__main-section--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-page__main-section--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-page__main-section--PaddingLeft: var(--pf-global--spacer--md);
+
+  background-color: #1a1a1a;
```

Comment:
I think the closes PF color to this one is `--pf-global--palette--black-900` can we use it here?